stdlib/tkinter/tix.pyi

```diff
@@ -45,7 +45,7 @@ class tixCommand:
     def tix_resetoptions(self, newScheme: str, newFontSet: str, newScmPrio: str | None = ...) -> None: ...
 
 class Tk(tkinter.Tk, tixCommand):
-    def __init__(self, screenName: str | None = ..., baseName: str | None = ..., className: str = ...): ...
+    def __init__(self, screenName: str | None = ..., baseName: str | None = ..., className: str = ...) -> None: ...
```

Comment:
The same arguments are now in 4 different places::
- `tkinter.Tcl()`
- `tkinter.Tk.__init__()`
- `tkinter.tix.Tk.__init__()`
- `ttkthemes.ThemedTk.__init__()`

It would be good to add comments to remind keeping all these in sync. For example, all but `tkinter.Tk.__init__` could have a comment `Please keep in sync with tkinter.Tk.__init__`, and then `tkinter.Tk.__init__` could have a comment that lists all other places where the same arguments are.

Reply:
Or even better than listing all the other places, it could recommend something like `git grep screenName`, which seems to find all the places nicely.